frappe/public/js/frappe/model/model.js

```diff
@@ -79,6 +79,10 @@ $.extend(frappe.model, {
 
 	init: function () {
 		// setup refresh if the document is updated somewhere else
+		if (frappe.socketio.is_document_listener_setup) {
```

Comment:
We can remove this for now, `model.init` is only called once and rn only does doc_update listener.